src/notification_manager/mod.rs

```diff
@@ -262,6 +270,20 @@ impl NotificationManager {
             "BOOLEAN",
             Some("false"),
         )?;
+        Self::add_column_if_not_exists(
+            db,
+            "user_info",
+            "hellthread_notifications_disabled",
+            "BOOLEAN",
+            None,
+        )?;
+        Self::add_column_if_not_exists(
+            db,
+            "user_info",
+            "hellthread_notifications_max_pubkeys",
+            "TINYINT",
```

Comment:
TINYINT should be -128 to 127, which should be more than enough for a setting like this.